lib/ember-cli/app.rb

```diff
@@ -237,7 +277,17 @@ def environment
     end
 
     def package_json
-      @package_json ||= JSON.parse(package_json_file_path.read).with_indifferent_access
+      @package_json ||=
+        JSON.parse(package_json_file_path.read).with_indifferent_access
+    end
+
+    def addon_package_json
+      @addon_package_json ||=
+        JSON.parse(addon_package_json_file_path.read).with_indifferent_access
+    end
+
+    def addon_version
+      addon_package_json.fetch("version", "0.0.0")
```

Comment:
Why do we need to specify the default value here? The user might have not defined it?